src/text-field-base.css.ts

```diff
@@ -0,0 +1,264 @@
+import {globalStyle, style} from '@vanilla-extract/css';
+import {sprinkles} from './sprinkles.css';
+import {vars} from './skins/skin-contract.css';
+import * as mq from './media-queries.css';
+
+const commonInputStyles = style([
+    sprinkles({
+        border: 'none',
+        minWidth: 0,
+        color: vars.colors.textPrimary,
+        width: '100%',
+    }),
+    {
+        background: 'none',
+        outline: 0,
+        fontSize: 'inherit',
+        lineHeight: 'inherit',
+        caretColor: vars.colors.controlActivated,
+        // Seems like 'display: flex' is causing issues on firefox and the input takes over the whole space https://stackoverflow.com/questions/43314921/strange-input-widths-in-firefox-vs-chrome
+        textOverflow: 'ellipsis',
+        selectors: {
+            '&::placeholder ': {
+                opacity: 0,
+                transition: 'opacity 150ms cubic-bezier(0.0, 0, 0.2, 1) 0ms',
+            },
+            '&:focus::placeholder ': {
```

Comment:
```suggestion
            '&:focus::placeholder': {
```

Reply:
we could place a rule to fix this automatically